src/components/Table/Table.story.jsx

```diff
@@ -1263,14 +1264,123 @@ storiesOf('Watson IoT|Table', module)
   .add('with zebra striping', () => (
     <Table useZebraStyles columns={tableColumns} data={tableData} actions={actions} />
   ))
+  .add('with resize and initial column widths on Simple Stateful and row selection', () => (
+    <StatefulTable
+      {...initialState}
+      actions={actions}
+      lightweight={boolean('lightweight', false)}
+      columns={tableColumns.map((i, idx) => ({
+        width: idx % 2 === 0 ? '100px' : '200px',
+        ...i,
+      }))}
+      options={{
+        hasRowSelection: select('hasRowSelection', ['multi', 'single'], 'multi'),
+        hasRowExpansion: false,
+        hasResize: true,
+      }}
+      view={{ table: { selectedIds: array('selectedIds', []) } }}
+    />
+  ))
+  .add(
+    'with resize and initial column widths and hidden column',
+    () => (
+      <FullWidthWrapper>
+        <Table
+          options={{ hasResize: true }}
+          columns={tableColumns.map((i, idx) => ({
+            width: idx % 2 === 0 ? '100px' : '200px',
+            ...i,
+          }))}
+          data={tableData}
+          actions={actions}
+          view={{
+            table: {
+              ordering: defaultOrdering,
+            },
+          }}
+        />
+      </FullWidthWrapper>
+    ),
+    {
+      info: {
+        source: true,
+        propTables: false,
+      },
+    }
+  )
+  .add(
+    'with resize and initial column widths',
+    () => (
+      <Table
+        options={{ hasResize: true }}
+        columns={tableColumns.map((i, idx) => ({
+          width: idx % 2 === 0 ? '100px' : '100px',
+          ...i,
+        }))}
+        data={tableData}
+        actions={actions}
+      />
+    ),
+    {
+      info: {
+        source: true,
+        propTables: false,
+      },
+    }
+  )
+  .add(
+    'with resize and no initial column width',
+    () => (
+      <Table
+        options={{ hasResize: true }}
+        columns={tableColumns}
+        data={tableData}
+        actions={actions}
+      />
+    ),
+    {
+      info: {
+        source: true,
+        propTables: false,
+      },
+    }
+  )
+  .add(
+    'with resize and no initial column width and auto adjusted column widths',
+    () => (
+      <React.Fragment>
+        <p>
+          <strong>Note!</strong> <br />
+          For this configuration to work, the table must be wrapped in a container that has a with
+          defined in other than %. <br />
+          E.g. the FullWidthWrapper used by the storybook examples.
+        </p>
+        <FullWidthWrapper>
+          <Table
+            options={{ hasResize: true, useAutoTableLayoutForResize: true }}
+            columns={tableColumns}
+            data={tableData}
+            actions={actions}
+          />
+        </FullWidthWrapper>
+      </React.Fragment>
+    ),
+    {
+      info: {
+        source: true,
+        propTables: false,
+      },
+    }
+  )
   .add(
```

Comment:
I'm not sure. This story existed before the resize functionality but I think it was then called 'with fixed column width'. I thought it would make sense to keep it.